src/App.tsx

```diff
@@ -1,26 +1,250 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
 
-const USER_ID = 0;
+import * as todoService from './api/todos';
+import { UserWarning } from './components/UserWarning';
+import { TodoHeader } from './components/TodoHeader';
+import { TodoList } from './components/TodoList';
+import { TodoFooter } from './components/TodoFooter';
+import { TodoNotification } from './components/TodoNotification';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [filterField, setFilterField] = useState('all');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+
+  const areAllTodosCompleted = () => {
+    const completedTodos = [...todos].filter(todo => todo.completed);
+
+    return completedTodos.length === todos.length;
+  };
+
+  useEffect(() => {
+    if (errorMessage) {
+      const timer = setTimeout(() => setErrorMessage(''), 3000);
+
+      return () => clearTimeout(timer);
+    }
+  }, [errorMessage]);
+
+  function loadTodos() {
```

Comment:
It is important to stick to one way of creating functions, fix it everywhere
```suggestion
  const loadTodos = () => {
```